Source/Facebook.Tests/HttpHelperTests.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Text;
+using System.Threading;

```

Comment:
Sorry, I might have tested something with async tasks and forgot to delete this line again.